app/Extensions/WoocommerceAfterpay.php

```diff
@@ -22,7 +22,7 @@ public function __construct() {
 	public function woocommerce_is_afterpay_enabled( $additional_settings ) {
 		if ( $woocommerce_afterpay_settings = get_option( 'woocommerce_afterpay_settings' ) ) {
 			if ( $woocommerce_afterpay_settings['enabled'] === 'yes' ) {
-				$additional_settings['woocommerce_is_afterpay_enabled'] = true;
+				$additional_settings['woocommerce_is_afterpay_enabled'] = "true";
```

Comment:
i dont know if i can accept this 😅

Reply:
let's reject it haha

Reply:
On second thought, let's have it merged because of a new issue: https://blazeonline.slack.com/archives/C05021WA3CZ/p1724039137994829?thread_ts=1724034728.474159&cid=C05021WA3CZ

Reply:
`The types of 'header.layout.desktop' are incompatible between these types.
    Type 'boolean' is not comparable to type 'string'.ts`
    
    values are "1" but these shouldn't be boolean